### PR TITLE
feat(experiment): adding faq and troubleshooting docs

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubham chaudhary <shubham@chaosnative.com>

## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: mkdocs/docs/experiments/faq/install.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+   /* Hide table of contents */
+      .md-sidebar--secondary {
+        display: none;
+      }
+  @media only screen and (min-width: 60em) {
+  .md-content {
+    margin: 0;
+  }
+  }
+</style>
```
Comment (uditgaurav) [inline mkdocs/docs/experiments/faq/install.md:19]: will this work with md file?

File: mkdocs/docs/experiments/troubleshooting/experiments.md [hunk 2]
```diff
@@ -198,4 +169,104 @@ kubectl describe chaosresult  -n <namespace>`
  
 This can occur sometimes due to the time taken in pulling the image starting the experiment pod (note that the ChaosResult resource is generated by the experiment). 
 For the above commands to execute successfully, you should simply wait for the experiment pod to be created. The waiting time will be based upon resource available
-(network bandwidth, space availability on the node filesyste
\ No newline at end of file
+(network bandwidth, space availability on the node filesyste
+
+### The helper pod is getting in a failed state due to container runtime issue
+
+??? info "View the error message"
```
Comment (uditgaurav) [inline mkdocs/docs/experiments/troubleshooting/experiments.md:176]: some typo?

Comment (ispeakc0de) [reply mkdocs/docs/experiments/troubleshooting/experiments.md:176]: what typo?

Comment (ispeakc0de) [reply mkdocs/docs/experiments/troubleshooting/experiments.md:176]: this is to add collapsable text

**Review by uditgaurav — APPROVED** [review]

File: mkdocs/docs/experiments/faq/install.md (hunk 1 above)
Comment (ispeakc0de) [reply mkdocs/docs/experiments/faq/install.md:19]: yes, it will work. But we are overriding this so removing this css

**Pull request merged**